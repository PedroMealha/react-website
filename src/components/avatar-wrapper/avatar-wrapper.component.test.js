import { shallow } from "enzyme";
import AvatarWrapper from "./avatar-wrapper.component";

describe("AvatarWrapper component", () => {
	it("expect to render", () => {
		const wrapper = shallow(<AvatarWrapper />)
		expect(wrapper).toMatchSnapshot();
	});
});
