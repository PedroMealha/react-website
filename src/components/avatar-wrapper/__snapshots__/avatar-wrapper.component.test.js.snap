// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AvatarWrapper component expect to render 1`] = `
<div
  className="avatar-wrapper"
>
  <div
    className="avatar"
  >
    <div
      className="avatar-picture"
    />
    <p
      className="name"
    >
      Pedro Mealha
    </p>
    <p
      className="profession"
    >
      Frontend Developer
    </p>
    <Link
      className="download"
      download={true}
      target="_blank"
      to="/CV_Pedro_Mealha.pdf"
    >
      <button>
        Download CV
      </button>
    </Link>
  </div>
  <div
    className="avatar-shadow"
  />
</div>
`;
